src/NServiceBus.Core/Timeout/Hosting/Windows/TimeoutPersisterReceiver.cs

```diff
@@ -72,63 +82,53 @@ void Poll(object obj)
         {
             var cancellationToken = (CancellationToken)obj;
 
-            var startSlice = DateTime.UtcNow.AddYears(-10);
+            var startSlice = currentTime().AddYears(-10);
```

Comment:
not a blocker, but when returning DateTime's, the naming should be clear about what timezone the returned DateTime is representing.